oaknational/Oak-Web-Application
# fix: use correct field names in highlight query [LESQ-528]

## Description

Music year: 2017

- update fields for highlight in the 2023 query to use correct casing
- this will need to be replicated in the search api when we switch to constructing the query there

## How to test

1. Go to https://deploy-preview-2218--oak-web-application.netlify.thenational.academy
2. Search for eg 'macbeth'
3. You should see the new results contain the keyword highlighted in bold

## Screenshots

How it used to look (delete if n/a):
<img width=500 src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/e17fa6e8-c993-4952-82d8-5a0b9e6e0216">


How it should now look:
<img width=500 src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/f1329075-cb9c-48c5-be21-347d8ac8aadd">

## Checklist
- [ ]  The matched search term should be in bold for any returned result
